example/model/model.ipynb

```diff
@@ -68,8 +68,8 @@
     "from dotenv import load_dotenv\n",
     "from IPython.display import display\n",
     "\n",
-    "from uniflow.client import Client\n",
-    "from uniflow.flow.flow_factory import FlowFactory\n",
+    "from uniflow.model.client import Client\n",
```

Comment:
same as above. model should not have a client interface. model should be used by both extract and transform.

Reply:
Updated with refactor commit